Collections/Grapple/grapple.alias

```diff
@@ -3,7 +3,7 @@ embed
 # Baseline Variables
 args     = argparse(&ARGS&)
 c        = combat()
-grappler = args.last("c", name)
+grappler = args.last("c", name if not c or not c.current or c.me else c.current.name)
```

Comment:
Should this `not (c or (c.current or c.me))` or something? the `or c.me` is throwing me off

Reply:
`not c or not (c.current or c.me)`

Reply:
The `c.me` is to preserve current behaviour if your active character is in combat.

In plain English... Use the active character if the active character is in initiative, or if there's no current combatant. Use current combatant otherwise.

Reply:
Maybe it would be clearer the other way round? `c.current.name if c and c.current and not c.me else name`

Reply:
Nah your explanation made sense to me I just was derping haha